mne/time_frequency/psd.py

```diff
@@ -95,9 +105,14 @@ def psd_array_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
 
     Returns
     -------
-    psds : ndarray, shape (..., n_freqs) or
+    psds : ndarray, shape (..., n_freqs)
         The power spectral densities. All dimensions up to the last will
-        be the same as input.
+        be the same as input unless combine is None. When combine is None
+        and input data ndim > 1 the -3 dimension corresponds to welch windows
+        (for example n_windows x n_channels x n_freqs or n_epochs x n_windows
+        x n_channels x n_freqs or when input data was n_epochs x n_times:
+        n_windows x n_epochs x n_times). For 1d input data the output is
+        n_windows x n_freqs if combine is None.
```

Comment:
I decided to give some examples, including the rare `n_epochs x n_times` input case. Hope it is not too convoluted.